firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -1972,6 +1974,22 @@ class BrowserViewController: UIViewController,
         presentSheetWith(viewModel: viewModel, on: self, from: button)
     }
 
+    func presentNewTabLongPressActionSheet(from view: UIView) {
+        let oneTapNewTabEnabled = featureFlags.isFeatureEnabled(.toolbarOneTapNewTab, checking: .buildOnly)
```

Comment:
should we also add an extra case for the toolbar refactor? I am assuming that we don't want this show unless toolbar refactor is also enabled. Or am I misinterpreting and is this independent?

Reply:
For iPhone: 
- We shouldn't even need this check for `one_tap_new_tab` since this code is only reachable when both flags are enabled. `one_tap_new_tab` is checked in [ToolbarMiddleware](https://github.com/mozilla-mobile/firefox-ios/blob/be3fe148af5164b5120f3dfc424295b3128e1ae0/firefox-ios/Client/Frontend/Browser/Toolbars/Redux/ToolbarMiddleware.swift#L571) and the middleware is only utilized when the toolbar refactor flag is enabled. The "new tab" button only displays when both flags are enabled as well.

for iPad:
- We check that `one_tap_new_tab` is enabled [before adding the long press gesture recognizer](https://github.com/mozilla-mobile/firefox-ios/pull/21287/files#diff-c9d9e9f873cd1167ac42d8dfab38f7052fb26a5e54c427c73b06fd7363b7dddcR74). We don't currently check for the toolbar refactor flag here... but maybe we should? iPad already has a "new tab" button that pre-dates the toolbar refactor so maybe it's ok to only check `one_tap_new_tab`

tagging @thatswinnie to make sure my understanding is correct (and verify the iPad notion) 🙂

Reply:
ah thank you for the context, in that case, it seems the feature flag should live at a higher level. From the latest changes, I see that the feature flag was removed, so will resolve this.